spec/ios_lint_localizations_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'spec_helper.rb'
+require 'fileutils'
+require 'tmpdir'
+require 'yaml'
+
+describe Fastlane::Actions::IosLintLocalizationsAction do
+  before do
+    # Ensure `Action.sh` is not skipped during test – so that SwiftGen will be installed by our action as normal
+    # See https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/helper/sh_helper.rb#L45-L85
```

Comment:
How do you feel about adding the commit hash version of this URL, so that it'll always point at the code mentioned as it was at the time of writing.

https://github.com/fastlane/fastlane/blob/e6bd288f17038a39cd1bfc1b70373cab1fa1e173/fastlane/lib/fastlane/helper/sh_helper.rb#L45-L85

I always have mixed feelings about using the commit hash URL version. It's great for the reason above, but it can also lead to confusion if the reader doesn't realized they're looking at code that might not be up to date anymore when triaging an issue.